rpm-builder/configure-validator.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+if /opt/ripple/bin/rippled -q server_info | \
+  grep -q 'no response from server'
+then
+  echo "rippled server is not running"
+  exit 1
+fi
+
+# Check for validation key
+VALIDATION_PUBLIC_KEY=`/opt/ripple/bin/rippled server_info -q | \
+  python -c 'import json,sys;obj=json.load(sys.stdin); \
+  print obj["result"]["info"]["pubkey_validator"]'`
```

Comment:
isn't it possible this raises an uncaught error if the response to server_info isn't formatted correctly?